src/components/AllGames/GameCardSlider/GameCardSlider.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
```

Comment:
Looks a bit strange, GameCardSlider is identical to usual GameCard, suggest using only GameCard.

Reply:
These cards different in styles